mobile/evidence/ios/Features/Tests/ProfileTests/ProfileTests.swift

```diff
@@ -34,24 +34,30 @@ final class ProfileTests: XCTestCase {
     func testClearStatusInputTextFieldSuccess() {
         let model = StatusViewModel()
         model.statusInput = "Some input"
-        model.clearStatusInputTextField()
+        model.clearStatusInputTextFieldTapped()
         XCTAssertTrue(model.statusInput.isEmpty)
         XCTAssertFalse(model.showClearButton)
     }
     
+    func testShowClearButtonAndCloseModalSuccess() {
+        let model = StatusViewModel()
+        model.showClearButtonAndCloseModal()
```

Comment:
Is this method called by the view? It seems to me that it is only called internally by other methods in the VM, so you should test them as they are executed by other methods. If this is called by the `saveButtonTapped` method, the assertions in this test should be moved to the test where you verify `saveButtonTapped` stuff.